k8s-install/scripts/install-cni.sh

```diff
@@ -116,29 +115,31 @@ KUBE_CA_FILE=${KUBE_CA_FILE:-$SERVICE_ACCOUNT_PATH/ca.crt}
 SKIP_TLS_VERIFY=${SKIP_TLS_VERIFY:-false}
 # Pull out service account token.
 SERVICEACCOUNT_TOKEN=$(cat $SERVICE_ACCOUNT_PATH/token)
+# TLS_CFG needs to be set regardless (to avoid error when creating calico-kubeconfig)
+TLS_CFG=""
 
 # Check if we're running as a k8s pod.
 if [ -f "$SERVICE_ACCOUNT_PATH/token" ]; then
   # We're running as a k8d pod - expect some variables.
-  if [ -z ${KUBERNETES_SERVICE_HOST} ]; then
+  if [ -z "${KUBERNETES_SERVICE_HOST}" ]; then
     echo "KUBERNETES_SERVICE_HOST not set"; exit 1;
   fi
-  if [ -z ${KUBERNETES_SERVICE_PORT} ]; then
+  if [ -z "${KUBERNETES_SERVICE_PORT}" ]; then
     echo "KUBERNETES_SERVICE_PORT not set"; exit 1;
   fi
 
-  if [ "$SKIP_TLS_VERIFY" == "true" ]; then
+  if [ "$SKIP_TLS_VERIFY" = "true" ]; then
     TLS_CFG="insecure-skip-tls-verify: true"
   elif [ -f "$KUBE_CA_FILE" ]; then
-    TLS_CFG="certificate-authority-data: $(cat $KUBE_CA_FILE | base64 | tr -d '\n')"
+    TLS_CFG="certificate-authority-data: $(base64 "$KUBE_CA_FILE" | tr -d '\n')"
```

Comment:
```suggestion
    TLS_CFG="certificate-authority-data: $(base64 -w 0 "$KUBE_CA_FILE")"
```

I was curious, I found a flag (`-w 0`) to disable line wrapping, which has a side-effect of not printing the ending (or any) newlines.

Reply:
Cool, yeah it's much clearer what the intention is.

Reply:
Done.